stress_test/sample_test_confs/beryllium/beryllium_nb_active_scalability_multinet.json

```diff
@@ -61,21 +61,21 @@
 "topology_get_switches_handler":"bin/handlers/get_switches",
```

Comment:
the sample test case parameters have to change back to their original values, don't they?